### PR TITLE
fix: add statement to approve the installplan when a new operator is being installed

### PR DESCRIPTION
**Pull request by CFSNM**
Related ticket: https://issues.redhat.com/browse/RHOAIENG-14362

Also enabling again the disruptive tests that were disabled in disconnected environments. This PR fixes that

Checks:

- Selfmanaged cluster OK
- Managed cluster OK
- Disconnected cluster OK

### CODE REVIEW

**Review by asanzgom — DISMISSED** [review]

File: ods_ci/tests/Resources/Page/Operators/ISVs.resource
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/Operators/ISVs.resource:35]: what if the upgade approval is not set to manual? We don't need to do this in that case. So it would be bettet to check the update approval first no?

Comment (CFSNM) [reply ods_ci/tests/Resources/Page/Operators/ISVs.resource:47]: yeah, changed the logic to just retrieve the installplan from the subscription and approve only if the approval is setted to manual.

Thanks @bdattoma for the review!

**Review by bdattoma — DISMISSED** [review]

**Comment by bdattoma** [comment]
consider backporting the PR after the merge :v:

**Comment by CFSNM** [comment]
@bdattoma yeah, once I merged that I plan to merge to the stable branches

**Comment by CFSNM** [comment]
@bdattoma can u please reapprove it? there was a typo in the IF clause that I had to fix

**Review by bdattoma — DISMISSED** [review]

**Review by bdattoma — APPROVED** [review]

**Pull request merged**
